### PR TITLE
Added documentation on how to get started with Dropwizard Archetype

### PR DESCRIPTION
**Pull request by jansoren**

### CODE REVIEW

**Comment by jansoren** [comment]
@jplock - could you please take a new (code)review? :-)

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -129,6 +127,23 @@ started.)
 .. _Leiningen: https://github.com/technomancy/leiningen
 .. __: http://www.sonatype.com/books/mvnref-book/reference/
 
+
+You have tree alternatives from here:
```
Comment (nickbabcock) [inline docs/source/getting-started.rst]: I'm guessing `tree` is a typo :smile:

Comment (jansoren) [inline docs/source/getting-started.rst]: Fixed

File: dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -42,6 +42,7 @@
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-core</artifactId>
+            <version>\${dropwizard.version}</version>
```
Comment (joschi) [inline dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/pom.xml]: This is unnecessary as the version if already imported via dropwizard-bom.

File: docs/source/getting-started.rst [hunk 3]
```diff
@@ -109,15 +109,13 @@ Now that you've gotten the lay of the land, let's dig in!
 
 .. _gs-maven-setup:
 
-Setting Up Maven
+Setting Up Using Maven
 ================
```
Comment (joschi) [inline docs/source/getting-started.rst]: Could you please fix the length of the line? Sphinx should give you related warnings.

File: dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+${name}
+==========
```
Comment (joschi) [inline dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/README.md]: Maybe switch to the alternate headings syntax of Markdown (`# ${name}`).

File: dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/pom.xml (hunk 2 above)
Comment (jansoren) [inline dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/pom.xml]: Fixed

File: docs/source/getting-started.rst (hunk 3 above)
Comment (jansoren) [inline docs/source/getting-started.rst]: Fixed

File: dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/README.md (hunk 4 above)
Comment (jansoren) [inline dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/README.md]: Fixed

**Comment by jplock** [comment]
LGTM, thanks!

**Pull request merged**